tests/aws/templates/cfn_lambda_logging_config.yaml

```diff
@@ -0,0 +1,53 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  FunctionName:
+    Type: String
+
+Resources:
+  MyFnServiceRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+        Version: "2012-10-17"
+      ManagedPolicyArns:
+        - Fn::Join:
+            - ""
+            - - "arn:"
+              - Ref: AWS::Partition
+              - :iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+  LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Ref FunctionName
+      Code:
+        ZipFile: |
+          exports.handler = function(event) {
+            return {
+              statusCode: 200,
+              body: "Hello, World!" 
+            };
+          };
+      Role:
+        Fn::GetAtt:
+          - MyFnServiceRole
+          - Arn
+      Handler: index.handler
+      Runtime: nodejs18.x
+      LoggingConfig:
```

Comment:
Maintainability: Do we need a dedicated tests for each sub-feature (e.g., LoggingConfig) or is there an existing test case we can extend?

Imagine we have a minimal case for testing defaults and a "reference" case that attempts to cover as much as possible of the API surface.

Reply:
TBH, I thought about this and what is the preference/convention in general at LS. My initial approach was to adapt an existing test, then decided to create a dedicated one (because I wasn't sure which test was the right one to add this new condition too, and if I did find it, what if someone removed that test for some reason - though this case still happen). 
Good point. It's probably better to adopt an existing one, since having so many tests is great, but each of them will take its time in the pipeline, so better to reuse what we have 💯

Reply:
good choice 👏